content/en/docs/reference/glossary/trust-domain.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Trust Domain
+---
+
+A trust domain is a globally unique name that Istio uses to create all
+[identities](/docs/reference/glossary/#identity) within a mesh. Every mesh has
+an exclusive trust domain.
```

Comment:
Not true today...

Reply:
yeah, but that *is* the definition ... should we leave it for now? Maybe adding a note "coming soon"? :)

Reply:
change to "should have"?

Reply:
So `Every mesh should have an exclusive trust domain.`?

Yeah I think that works for now.

Reply:
I think we should try to avoid talking about things "coming soon" in the glossary (and in the docs in general). I like the "should have ..." suggestion.

Reply:
I agree with @frankbu regarding forward looking statements. If it is not true now, we can leave it out until it is and then make a PR to add it.

Reply:
@nmittler @smawson Would leaving out the sentence for now work?